src/MoBi.Presentation/MenusAndBars/ContextMenus/ContextMenuForSimulation.cs

```diff
@@ -40,10 +40,11 @@ public IContextMenu InitializeWith(IObjectBaseDTO dto, IPresenter presenter)
          {
             createEditItem(simulation),
             createRenameItem(simulation),
-            
 
-            createNewSimulationMenuBarItem(),
-            createAddExistingSimulationMenuBarItem(),
+            createConfigure(simulation),
+            
+//            createNewSimulationMenuBarItem(),
```

Comment:
@Yuri05  I am not sure WHY we have those guys where on a simuation? It does not make sense to be to have those menus at the Folder level and the simulation level, especially because there is absolutely no link with the selected simulation